sys/cheri/cherireg.h

```diff
@@ -116,8 +116,13 @@
 #define	CHERI_SEALCAP_KERNEL_OFFSET	0x0
 
 /* Reserved CHERI object types: */
+#ifdef __aarch64__
+#define	CHERI_OTYPE_UNSEALED	(0)
+#define	CHERI_OTYPE_SENTRY	(1l)
+#else
 #define	CHERI_OTYPE_UNSEALED	(-1l)
 #define	CHERI_OTYPE_SENTRY	(-2l)
+#endif
```

Comment:
These probably belong back in the MD headers.

Reply:
Or even better, we use cheriintrin.h which already defines those macros.

Reply:
But that might need some include path changes, since the kernel builds with -nostdinc?

Reply:
If you want to go fix the build system so we can reliably use compiler headers when building the base system then be my guest. We all know we should be using cheriintrin.h and saying it won't make the blocking issue go away :)

Reply:
Yeah unfortunately that change is quite painful. Maybe we can just symlink the builtin header to WORLDTMP?